testfiles/UnitarySystem_MultiSpeedDX.idf

```diff
@@ -2699,8 +2707,8 @@
     Sys 1 Furnace DX Cool Air Loop Inlet,  !- Return Air Node Name
     Sys 1 Furnace DX Cool Mixed Air Outlet,  !- Mixed Air Node Name
     Sys 1 Furnace DX Cool Outdoor Air Inlet,  !- Actuator Node Name
-    autosize,                !- Minimum Outdoor Air Flow Rate {m3/s}
-    autosize,                !- Maximum Outdoor Air Flow Rate {m3/s}
+    0.0,                !- Minimum Outdoor Air Flow Rate {m3/s}
+    0.0,                !- Maximum Outdoor Air Flow Rate {m3/s}
```

Comment:
I assume you are trying to target another configuration that we don't have yet with these type of changes to idfs.

Reply:
No, these changes were supposed to remain local. Will revert this and a couple of others.